frontend/javascripts/oxalis/view/components/input_component.tsx

```diff
@@ -103,20 +95,34 @@ class InputComponent extends React.PureComponent<InputComponentProps, InputCompo
 
   render() {
     const { isTextArea, onPressEnter, title, style, ...inputProps } = this.props;
-    const InputClass = isTextArea ? Input.TextArea : Input;
-    const defaultOnPressEnter = !isTextArea ? this.blurYourself : null;
-    const input = (
-      <InputClass
-        {...inputProps}
-        style={title == null ? style : null}
-        onChange={this.handleChange}
-        onFocus={this.handleFocus}
-        onBlur={this.handleBlur}
-        value={this.state.currentValue}
-        onPressEnter={onPressEnter != null ? onPressEnter : defaultOnPressEnter}
-        onKeyDown={this.blurOnEscape}
-      />
-    );
+    let input;
+    if (isTextArea) {
```

Comment:
Why was it needed to split this up and duplicate the instantiation code?

Reply:
I think I introduced this if/else in the typescript pr because typescript was complaining about something and this if/else fixed the error.

After a closer look, I could fix the problem now :)

Reply:
Awesome, thanks for having another look :+1: